.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
   pc-emul:
     runs-on: self-hosted
-    timeout-minutes: 5
+    timeout-minutes: 40
```

Comment:
keep `timeout-minutes: 5`
nix-dependencies are installed on the first time the nix-shell is activated only.
The current nix setup in self-hosted machines caches nix packages for all users.
When adding a new nix package dependecy, go to each self-hosted machine and activate the nix-shell to install the dependency for all users.

Reply:
ok will keep it at 5, i putted at 40 so it could install the magic in the nix